File: Sem4/MPP/Lab/ProiectCSharp/model/Excursie.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace model
{
    public class Excursie : Entity<int>
    {
        public String obiectivVizitat { get; set; }
        public DateTime oraPlecare { get; set; }
        public String firmaTransport { get; set; }
        public Double pret { get; set; }
        public int locuriDisponibile { get; set; }

        public int id { get; set; }

        public Excursie(string obiectivVizitat, DateTime oraPlecare, string firmaTransport, double pret, int locuriDisponibile)
        {
            this.obiectivVizitat = obiectivVizitat;
            this.oraPlecare = oraPlecare;
            this.firmaTransport = firmaTransport;
            this.pret = pret;
            this.locuriDisponibile = locuriDisponibile;
        }

        public Excursie()
        {
            this.obiectivVizitat = null;
            this.oraPlecare = DateTime.MinValue;
            this.firmaTransport = null;
            this.pret = 0;
            this.locuriDisponibile = 0;
        }

        public void setId(int id)
        {
            this.id = id;
        }

        public int getId()
        {
            return this.id;
        }
    }
}

File: Sem4/MPP/Lab/ProiectJava/ProjectModel/src/main/java/project/model/Excursie.java
package project.model;

import java.time.LocalDateTime;

public class Excursie implements Entity<Integer> {
	private String obiectivVizitat;
	private LocalDateTime oraPlecare;
	private String firmaTransport;
	private Double pret;
	private int locuriDisponibile;
	private int id;

	public Excursie(String obiectivVizitat, LocalDateTime oraPlecare, String firmaTransport, Double pret, int locuriDisponibile) {
		this.obiectivVizitat = obiectivVizitat;
		this.oraPlecare = oraPlecare;
		this.firmaTransport = firmaTransport;
		this.pret = pret;
		this.locuriDisponibile = locuriDisponibile;
	}

	public Excursie() {
		this.id=0;
		this.obiectivVizitat = null;
		this.oraPlecare = null;
		this.firmaTransport = null;
		this.pret = null;
		this.locuriDisponibile = 0;
	}

	public String getObiectivVizitat() {
		return obiectivVizitat;
	}

	public void setObiectivVizitat(String obiectivVizitat) {
		this.obiectivVizitat = obiectivVizitat;
	}

	public LocalDateTime getOraPlecare() {
		return oraPlecare;
	}

	public void setOraPlecare(LocalDateTime oraPlecare) {
		this.oraPlecare = oraPlecare;
	}

	public String getFirmaTransport() {
		return firmaTransport;
	}

	public void setFirmaTransport(String firmaTransport) {
		this.firmaTransport = firmaTransport;
	}

	public Double getPret() {
		return pret;
	}

	public void setPret(Double pret) {
		this.pret = pret;
	}

	public int getLocuriDisponibile() {
		return locuriDisponibile;
	}

	public void setLocuriDisponibile(int locuriDisponibile) {
		this.locuriDisponibile = locuriDisponibile;
	}

	@Override
	public void setId(Integer id) {
		this.id=id;
	}

	@Override
	public Integer getId() {
		return id;
	}

	@Override
	public String toString() {
		return "Excursie{" +
				"obiectivVizitat='" + obiectivVizitat + '\'' +
				", oraPlecare=" + oraPlecare +
				", firmaTransport='" + firmaTransport + '\'' +
				", pret=" + pret +
				", locuriDisponibile=" + locuriDisponibile +
				", id=" + id +
				'}';
	}
}
